build-system/server/typescript-compile.js

```diff
@@ -14,29 +14,38 @@
  * limitations under the License.
  */
 const log = require('fancy-log');
+const pathModule = require('path');
 const {cyan, green} = require('ansi-colors');
 const {exec} = require('../common/exec');
 
 const SERVER_TRANSFORM_PATH = 'build-system/server/new-server/transforms';
 
-// Used by new server implementation
-const typescriptBinary = './node_modules/typescript/bin/tsc';
+function getBuildCmd() {
+  switch (process.platform) {
+    case 'win32':
+      return `node .\\node_modules\\typescript\\lib\\tsc.js -p ${SERVER_TRANSFORM_PATH.split(
+        '/'
+      ).join(pathModule.sep)}${pathModule.sep}tsconfig.json`;
+
+    default:
+      return `./node_modules/typescript/bin/tsc -p ${SERVER_TRANSFORM_PATH}/tsconfig.json`;
+  }
+}
 
 /**
  * Builds the new server by converting typescript transforms to JS
  */
 function buildNewServer() {
-  const buildCmd = `${typescriptBinary} -p ${SERVER_TRANSFORM_PATH}/tsconfig.json`;
   log(
     green('Building'),
     cyan('AMP Dev Server'),
     green('at'),
     cyan(`${SERVER_TRANSFORM_PATH}/dist`) + green('...')
   );
-  const result = exec(buildCmd, {'stdio': ['inherit', 'inherit', 'pipe']});
+  const result = exec(getBuildCmd(), {'stdio': ['inherit', 'inherit', 'pipe']});
   if (result.status != 0) {
     const err = new Error('Could not build AMP Dev Server');
-    err.showStack = false;
+    err.showStack = true;
```

Comment:
The reason this was `false` in the first place was because the message in `err` contained the useful info, while the stack contained a very verbose and mostly useless nodejs callstack. Has any of this changed? (If yes, you can delete this line, since the default is true. If not, I'd suggest reverting this line.)